api/hasura/actions/_handlers/createUserWithToken.ts

```diff
@@ -141,6 +145,20 @@ const handleOrg = async (
 
   const org = org_share_tokens[0]?.organization;
   if (!org) return false;
+  if (org_share_tokens[0]?.type != ShareTokenType.Invite) {
+    // if this is just a welcome link, we need to see if they are a guild member, otherwise they can't join
```

Comment:
`  if (org_share_tokens[0]?.type != ShareTokenType.Invite)` if the share type is not an invite, does that mean its a welcome link?

I would expect this to be `if share_token_type == welcome link` not is not Invite... ?